net/Cargo.toml

```diff
@@ -10,6 +10,8 @@ tempfile = "=3.0.7"
 [dependencies]
 directories = "=1.0.2"
 failure = "=0.1.5"
+holochain_lib3h = { path = "../../lib3h/lib3h" }
+holochain_lib3h_protocol = { path = "../../lib3h/lib3h_protocol" }
```

Comment:
I don't think we can merge this until we publish a version of these to crates.io and put version numbers here

Reply:
yep. No worries. CI is red all over.